includes/admin/OrderSettings.php

```diff
@@ -291,12 +292,12 @@ private function getAgeCheckOfProduct(): ?bool
 
             $productAgeCheck = WCX_Product::get_meta($product, WCMYPA_Admin::META_AGE_CHECK, true);
 
-            if ($productAgeCheck === WCMYPA_Admin::PRODUCT_OPTIONS_ENABLED) {
-                return true;
+            if (empty($productAgeCheck)) {
+                $hasAgeCheck = WCMYPA_Admin::PRODUCT_OPTIONS_DEFAULT;
             }
```

Comment:
dit kan toch een else if zijn?

Reply:
Dat kan zeker. Alleen vind ik gebruik van een else niet zo mooi.

Reply:
Ik zou het ook niet doen als je iets returned ofzo, maar hier vind ik het leesbaarder. Nu zat ik te zoeken of de if statements allebei tegelijk true zouden zijn, maar met een else/elseif weet je meteen dat dat nooit kan